tests/e2e/vsphere.go

```diff
@@ -1049,3 +1049,47 @@ func (vs *vSphere) verifyVolumeCompliance(volumeID string, shouldBeCompliant boo
 		gomega.Expect(queryResult.Volumes[0].ComplianceStatus == "compliant").To(gomega.BeFalse())
 	}
 }
+
+// verifyLabelsAreUpdated executes cns QueryVolume API on vCenter and verifies if
+// volume labels are updated by metadata-syncer
+func (vs *vSphere) verifyLabelsAreUpdated(volumeID string, matchLabels map[string]string,
```

Comment:
if you are reusing the code from `waitForLabelsToBeUpdated()`, then please change `waitForLabelsToBeUpdated()` to call `verifyLabelsAreUpdated()` in-turn so that we don't have to manage two parts of code doing the same thing

Reply:
Done